build/extract.js

```diff
@@ -240,6 +240,7 @@ class Extract {
             let base = stripMarks(char.normalize('NFD'));
             if (base !== char && base !== '') {
               lang.data[char] = {
+                capital: char.toLowerCase() !== char,
```

Comment:
The PR looks god to me. However, as you've also already noticed we have e.g. Arabic where there's always capital set to "false". I've googled around and found:

> There is no different form for uppercase letters and lowercase letters.

https://simple.wikipedia.org/wiki/Arabic_alphabet

Now, this leads to the question: What should happen in this case and how can this be determined at all? The answer to the first question is quite easy I think: The diacritic has no capital classification, therefore it would be e.g. included in both options, when "caseSensitive" is set to true and false (in the context of mark.js). So what I would expect from an output point of view would be e.g.:

```javascript
{"lowercase": true, "uppercase": true}
```

or something in that direction.

But how can it be determined? From my point of view by comparing if the character is equally when converting it to both -- toLowerCase() and toUpperCase()

What do you think?

Reply:
What about having one variable named "case" with three settings: "upper", "lower" and "none" (or "both")? I would lean towards "none" because in the "es" language we include punctuation characters like "¿", "¡" and "⸘".

Why do we need to check the character's upper and lower case values now that we require the `capital` value?

Reply:
> What about having one variable named "case" 

I'm fine with that.

> Why do we need to check the character's upper and lower case values now that we require the capital value?

For the extract process? Or am I missing something? Of course for manually created files you can create this property too.

Reply:
Oops, you're right. I wasn't thinking about the extract process.